docs/book/v3/features/template/laminas-view.md

```diff
@@ -18,164 +18,290 @@ To use the laminas-view wrapper, you must first install the laminas-view integra
+    'script_paths' => $namespacedPaths,
+    // ... Other Options
+]);
+
+/**
+ * Now we can create the aggregate resolver: 
+ */
+$aggregateResolver = new AggregateResolver([
+    $mapResolver,
+    $namespacedResolver,
+]);
+
+/**
+ * And locate template files: 
+ */
+$one = $aggregateResolver->resolve('one'); // templates/one.phtml
+$kermit = $aggregateResolver->resolve('muppets::kermit'); // templates/muppets/kermit.phtml
 ```
 
-> ### Namespaced path resolving
->
-> Mezzio defines a custom laminas-view resolver,
-> `Mezzio\LaminasView\NamespacedPathStackResolver`. This resolver
-> provides the ability to segregate paths by namespace, and later resolve a
-> template according to the namespace, using the `namespace::template` notation
-> required of `TemplateRendererInterface` implementations.
->
-> The `LaminasView` adapter ensures that:
->
-> - An `AggregateResolver` is registered with the renderer. If the registered
->   resolver is not an `AggregateResolver`, it creates one and adds the original
->   resolver to it.
-> - A `NamespacedPathStackResolver` is registered with the `AggregateResolver`, at
->   a low priority (0), ensuring attempts to resolve hit it later.
+The configuration above essentially describes the default setup you will get.
+
+All you need to do is configure the templates and retrieve the template resolver from the container:
+
+```php
+// config/autoload/templates.global.php
+return [
+    'templates' => [
+        'default_suffix' => 'phtml',
+        'map' => [
+            'one' => __DIR__ . '/templates/one.phtml',
+            'two' => __DIR__ . '/templates/two.phtml',
+        ],
+        'paths' => [
+            'muppets' => __DIR__ . '/templates/muppets',
+        ],
+    ],
+];
+```
+
+```php
+use Laminas\View\Resolver\ResolverInterface;
+use Psr\Container\ContainerInterface;
+
+assert($container instanceof ContainerInterface);
+
+$resolver = $container->get(ResolverInterface::class);
+
+$resolver->resolve('one'); // templates/one.phtml
+$resolver->resolve('muppets::kermit'); // templates/muppets/kermit.phtml
+$resolver->resolve('not-there'); // false
+```
+
+We encourage you to familiarise yourself with the [template resolver documentation](https://docs.laminas.dev/laminas-view/v3/template-resolvers/) in `laminas-view` for alternative ways of resolving templates and how to implement your own template resolver.
+
+> ### Namespaced path resolving in Mezzio
 >
-> With resolvers such as the `TemplateMapResolver`, you can also resolve
-> namespaced templates, mapping them directly to the template on the filesystem
-> that matches; adding such a resolver can be a nice performance boost!
+> The Mezzio-specific `Mezzio\LaminasView\NamespacedPathStackResolver` satisfies the convention of "namespaced" template paths.
+> The format is `namespace::template` and typically, `namespace` refers to one or more directories and `template` refers to a file named `template.phtml` in one of those directories.
```

Comment:
A simpler syntax can be used:

```markdown
NOTE: **Namespaced Path Resolving in Mezzio**
The Mezzio-specific `Mezzio\LaminasView\NamespacedPathStackResolver` satisfies the convention of "namespaced" template paths.
The format is `namespace::template` and typically, `namespace` refers to one or more directories and `template` refers to a file named `template.phtml` in one of those directories.
```

Reply:
Ah, OK! I can never remember the formatting for callouts 🤦‍♂️